tools/checkcomments/checkcomments_test.go

```diff
@@ -81,7 +81,7 @@ func TestClient(t *testing.T) {
 	t.Run("IssueStatus", func(t *testing.T) {
 		t.Parallel()
 
-		c, err := newClient(cacheFilePath, t.Logf, t.Logf)
+		c, err := newClient(cacheFilePath, t.Logf, t.Logf, t.Logf)
```

Comment:
Is this intentional or by accident?

Reply:
Well, intentional: `newClient` accepts one more logging function now